lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -59,7 +61,23 @@ export class RefreshTokenClient extends BaseClient {
             undefined,
             true,
             request.forceCache
-        );
+        ).then((result: AuthenticationResult) => {
+            atsMeasurement?.endMeasurement({
+                success: true,
+                refreshTokenSize: response.body.refresh_token?.length || 0
```

Comment:
The refresh token size is recorded when a refresh token is returned by acquireToken() and is taken as 0 in all other scenarios for consistency purposes.